enterprise/websvc.manager/src/org/netbeans/modules/websvc/manager/model/WebServiceData.java

```diff
@@ -216,16 +216,6 @@ public void setWsdlFile(String fileName) {
         this.wsdlFile = fileName;
     }
 
-    /**
-     * 
-     * @return the WSDL file (absolute path)
-     * @deprecated use {@link #getWsdlFile()} instead
-     */
-    @Deprecated
-    public String getURL() {
```

Comment:
This changes the API of the module. If you want to remove this, you'll have to
- check all current users of the module if the method is used (it is friend-only) and fix these
- you'll have to increment major version and update specification version (see: http://wiki.netbeans.org/VersioningPolicy)